code/modules/photography/camera/camera.dm

```diff
@@ -227,3 +227,69 @@
 		p.set_picture(picture, TRUE, TRUE)
 		if(CONFIG_GET(flag/picture_logging_camera))
 			picture.log_to_file()
+
+
+/obj/item/camera/syndicate/attack(mob/living/carbon/human/M, mob/user) // syndicate camera flash
+	if(!is_syndicate(user))
+		. = ..() // Non-syndicates get the normal camera usage when they try to use it.
+		return
+	if(!on)
+		return
+	if(iscarbon(M))
+		flash_carbon(M, user, 5, 1)
+		playsound(src, 'sound/weapons/flash.ogg', 100, TRUE)
+		flash_lighting_fx(8, light_power, light_color)
+		flash_cooldown(user)
+		return TRUE
+	else if(issilicon(M))
+		var/mob/living/silicon/robot/R = M
+		log_combat(user, R, "flashed", src)
+		R.Paralyze(rand(80,120))
+		var/diff = 5 * CONFUSION_STACK_MAX_MULTIPLIER - M.confused
+		R.confused += min(5, diff)
+		R.flash_act(affect_silicon = 1)
+		playsound(src, 'sound/weapons/flash.ogg', 100, TRUE)
+		flash_cooldown(user)
+		flash_lighting_fx(8, light_power, light_color)
+		user.visible_message("<span class='disarm'>[user] overloads [R]'s sensors with the camera flash!</span>", "<span class='danger'>You overload [R]'s sensors with the flash!</span>") // second message generic
+		return TRUE
+
+
+/obj/item/camera/syndicate/proc/flash_carbon(mob/living/carbon/M, mob/user, power = 15, targeted = TRUE, generic_message = TRUE)
+	if(!istype(M) || !on)
+		return
+	if(user)
+		log_combat(user, M, "[targeted? "flashed(targeted)" : "flashed(AOE)"]", src)
+	if(generic_message && M != user)
+		to_chat(M, "<span class='disarm'>[src] emits a blinding light!</span>")
+	if(targeted)
+		if(M.flash_act(1, 1))
+			if(M.confused < power)
+				var/diff = power * CONFUSION_STACK_MAX_MULTIPLIER - M.confused
+				M.confused += min(power, diff)
+			if(user)
+				to_chat(M, "<span class='userdanger'>You are blinded by [src]!</span>")
+			M.Paralyze(rand(80,120))
+		else if(user)
+			visible_message("<span class='disarm'>[user] fails to blind [M] with the flash!</span>") // generic flash message on purpose
+			to_chat(user, "<span class='warning'>You fail to blind [M] with the camera flash!</span>")
+			to_chat(M, "<span class='danger'>[user] fails to blind you with the flash!</span>") // generic flash message on purpose
+		else
+			to_chat(M, "<span class='danger'>[src] fails to blind you!</span>")
+	else
+		if(M.flash_act())
+			var/diff = power * CONFUSION_STACK_MAX_MULTIPLIER - M.confused
+			M.confused += min(power, diff)
+
+
+/obj/item/camera/syndicate/proc/flash_cooldown(mob/user)
+	var/realcooldown = cooldown + 3 SECONDS // longer delay to recharge than normal usage
+	if(user)
+		var/mob/living/carbon/human/H = user
+		if (HAS_TRAIT(H, TRAIT_PHOTOGRAPHER)) // yes; knowing how to use a camera will help you out here as well
```

Comment:
The 1 point perk giving direct combat upgrade for a signle traitor item as its defining characteristic

Reply:
> The 1 point perk giving direct combat upgrade for a signle traitor item as its defining characteristic

Your feedback is a bit confusing. Are you saying that I should remove this portion because it is bad or are you just pointing out this exists so people don't miss it?

Reply:
Please clarify @SomeguyManperson

Reply:
I don't think there's any other quirk which gives such a direct in combat buff especially not a 1 point one so yes it's stupid

Reply:
> I don't think there's any other quirk which gives such a direct in combat buff 

Yet